src/components/CCPDashboard/PatientDetailsDialog.tsx

```diff
@@ -67,7 +68,7 @@ export const PatientDetailsDialog = (props: PatientDetailsDialogProps) => {
   ];
 
   const handleCloseDetails = () => {
-    history.push(`/events/${eventId}/ccps/${ccpId}`);
+    history.push(`/events/${eventId}/ccps/${ccpId}`, { numUpdates });
```

Comment:
This might not make a difference at the moment, but I think it should be:
```suggestion
    history.push(`/events/${eventId}/ccps/${ccpId}`, { ...location.state, numUpdates });
```